tests/unit/s2n_random_test.c

```diff
@@ -792,6 +792,23 @@ static int s2n_random_rand_bytes_after_cleanup_cb(struct random_test_case *test_
     return S2N_SUCCESS;
 }
 
+static int s2n_random_rand_bytes_before_init(struct random_test_case *test_case)
+{
+#if S2N_LIBCRYPTO_SUPPORTS_CUSTOM_RAND
+    /* Calling RAND_bytes will set a global random method */
+    unsigned char rndbytes[16] = { 0 };
```

Comment:
Is there any way to verify this assumption? What happens if you call RAND_get_rand_method here?

Reply:
It depends on the version of Openssl you're using. Earlier versions will return RAND_SSLeay() and later versions will return RAND_OpenSSL(). I added an assert that it is not our custom rand method b/c the branch is kind of annoying.